### PR TITLE
adding kuberay-operator package

### PR DESCRIPTION
**Pull request by mauricio-dc-chainguard**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: kuberay-ray-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  - runs: |
+      # https://github.com/ray-project/kuberay/blob/master/ray-operator/DEVELOPMENT.md
+      export BUILD_TIME=$(date "+%F %T")
+      export COMMIT_SHA1=$(git rev-parse HEAD)
+      cd ray-operator
+      rm -rf bin
+      go fmt ./...
+      go vet ./...
+      go build -ldflags "-X 'main._buildTime_=$BUILD_TIME' -X 'main._commitId_=$COMMIT_SHA1'" -o bin/manager main.go
```
Comment (Dentrax) [inline kuberay-ray-operator.yaml:32]: you can use `go/build` pipeline here

File: kuberay-ray-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      tag: v${{package.version}}
+
+  - runs: |
+      # https://github.com/ray-project/kuberay/blob/master/ray-operator/DEVELOPMENT.md
+      export BUILD_TIME=$(date "+%F %T")
+      export COMMIT_SHA1=$(git rev-parse HEAD)
+      cd ray-operator
+      rm -rf bin
+      go fmt ./...
+      go vet ./...
```
Comment (Dentrax) [inline kuberay-ray-operator.yaml:31]: we don't need those actually

File: kuberay-ray-operator.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      go build -ldflags "-X 'main._buildTime_=$BUILD_TIME' -X 'main._commitId_=$COMMIT_SHA1'" -o bin/manager main.go
+
+  - uses: strip
+
```
Comment (Dentrax) [inline kuberay-ray-operator.yaml:35]: we can add simple `test:` pipeline here and run something like `/usr/bin/manager --help` or `/usr/bin/manager version`.

Comment (mauricio-dc-chainguard) [reply kuberay-ray-operator.yaml]: added testing and successful results locally

**Review by Dentrax — DISMISSED** [review]

File: kuberay-operator.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    strip-prefix: v
+    tag-filter: v1.0.0
+
+test:
+  environment:
+    contents:
+      repositories:
+        - https://packages.wolfi.dev/os
+      keyring:
+        - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```
Comment (ajayk) [inline kuberay-operator.yaml:46]:
```suggestion
```

File: kuberay-operator.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      packages:
+        - wolfi-base
+        - kuberay-operator
```
Comment (ajayk) [inline kuberay-operator.yaml:49]:
```suggestion
```

**Review by ajayk — APPROVED** [review]

**Pull request merged**